cms/src/main/java/uk/nhs/digital/apispecs/swagger/CodegenDefaultGenerator.java

```diff
@@ -0,0 +1,51 @@
+package uk.nhs.digital.apispecs.swagger;
+
+import static freemarker.template.utility.Collections12.singletonList;
+import static java.util.stream.Collectors.toList;
+
+import io.swagger.codegen.v3.CodegenOperation;
+import io.swagger.codegen.v3.DefaultGenerator;
+import io.swagger.v3.oas.models.Paths;
+import io.swagger.v3.oas.models.tags.Tag;
+
+import java.util.*;
+
+/**
+ * Some producer teams require ordering their endpoints in a non-lexographical order.
+ * This requires a custom extension of the DefaultGenerator provided by CodeGen.
+ *
+ * This class CodegenDefaultGenerator extends DefaultGenerator class and sorts CodegenOperations before returning them.
+ * See the method orderedByDeclaredTags.
+ */
+public class CodegenDefaultGenerator extends DefaultGenerator {
```

Comment:
Would you mind adding this class to the compile list in `ApiSpecificationRenderer.groovy` and make sure the updater script works, Harry?

Reply:
yep, compiles without issues